content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,159 @@
+
+.. _viva_wallet/identifier:
+
+Terminal ID
+~~~~~~~~~~~
+
+To find your terminal ID number:
+
+#. Go to your Viva Wallet account and select the relevant account.
+#. Go to :menuselection:`Sales --> Physical payments --> Card terminals` in the navigation menu.
+
+The terminal ID number is located under the :guilabel:`Terminal ID (TID)` column.
+
+.. image:: viva_wallet/terminal-id.png
+   :alt: Viva terminal ID
+
+.. _viva_wallet/method-creation:
+
+Configure the payment method
+----------------------------
+
+Enable the payment terminal :ref:`in the application settings <configuration/settings>` and
+:doc:`create the related payment method <../../payment_methods>`. Set the journal type as
+:guilabel:`Bank` and select :guilabel:`Viva Wallet` in the :guilabel:`Use a Payment Terminal` field.
+
+Finally, fill in the mandatory fields with your
+
+- :ref:`Merchant ID and API key <viva_wallet/ID-key>`
+- :ref:`Client ID and Client secret <viva_wallet/pos-apis>`
+- :ref:`Terminal ID <viva_wallet/identifier>`
+
+.. image:: viva_wallet/create-method-viva-wallet.png
+   :alt: payment method creation form
+   :scale: 75%
+
+Then, save the form and copy the generated webhook URL of the :guilabel:`Viva Wallet Webhook
+Endpoint` field.
+
+Configure the Webhook
+---------------------
+
+Webhooks allow you to receive real-time notifications whenever a transaction occurs within your Viva
+Wallet account. To set them up:
+
+#. Go to :menuselection:`Settings --> API Access --> Webhooks` in your Viva Wallet account.
+#. Click :guilabel:`Create Webhook`.
+
+   .. image:: viva_wallet/register-webhook.png
+      :alt: webhook creation page
+
+#. Enter the previously generated Webhook URL in the :guilabel:`URL` field. Click :guilabel:`Verify`
+   to confirm its validity.
+#. From the :guilabel:`Event Type` dropdown menu, select :guilabel:`Transaction Payment Created`.
+#. Enable notifications by checking the :guilabel:`Active` checkbox.
+#. Click :guilabel:`Save` to complete the configuration.
+
+   .. image:: viva_wallet/webhook-form.png
+      :alt: webhook creation popup window
+
+.. seealso::
+   - `Setting up webhooks <https://developer.viva.com/webhooks-for-payments/#setting-up-webhooks>`_
+   - `Webhooks for payment transactions <https://developer.vivawallet.com/webhooks-for-payments/transaction-payment-created/>`_
+
+Link the payment method to a POS
+--------------------------------
+
+You can select the payment method in your POS settings once the payment method is created. To do so,
+go to the :ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment
+method under the :guilabel:`Payment methods` field of the :guilabel:`Payments` section.
+
+Pay with a payment terminal
+===========================
+
+When processing a payment, select the related payment method. Check the amount and
+click on :guilabel:`Send`. Once the payment is successful, the status changes to :guilabel:`Payment
+Successful`.
```

Comment:
Not sure this section is necessary. Could we just add a see also to the generic documentation about payment terminals?

Reply:
This is a good suggestion. However, this section appears in most (if not all) terminals' documentation pages. I've created a task to deal with this altogether  :)